Documentation/kubelet-wrapper.md

```diff
@@ -27,21 +27,37 @@ ExecStart=/usr/lib/coreos/kubelet-wrapper \
 
 In the example above we set the `KUBELET_VERSION` and the kubelet-wrapper script takes care of running the correct container image with our desired API server address and manifest location.
 
+### Customizing rkt Options
+
+Passing customized options or flags to rkt can be accomplished with the RKT_OPTS environment variable. Referencing it in a unit file is straightforward. Example below mounts host's `/etc/resolv.conf` file directly into the container:
+
+```ini
+[Service]
+Environment=RKT_OPTS="--volume=resolv,kind=host,source=/etc/resolv.conf --mount volume=resolv,target=/etc/resolv.conf"
```

Comment:
`--dns` will be fixed in 1.4. Please check the issue: https://github.com/coreos/rkt/issues/2141
And initial commit contained info about `please use rkt greater than 1.2 (>1.2, not >=1.2)`. Them milestone was changed from 1.3 to 1.4 and I decided to completely remove `--dns`.